pandas/core/generic.py

```diff
@@ -8865,16 +8871,27 @@ def mask(self, cond, other=np.nan, inplace=False, axis=None, level=None,
         2   NaN  15.0  18.0
         3   NaN  30.0  33.0
         4   NaN  45.0  48.0
+        
+        .. versionchanged:: 0.24.0
```

Comment:
Can remove this directive and add a sentence like You can specify the value to fill with.

Reply:
done